sklearn/metrics/tests/test_pairwise_distances_reduction.py

```diff
@@ -660,6 +660,124 @@ def test_argkmin_factory_method_wrong_usages():
         ArgKmin.compute(X=X, Y=Y, k=k, metric=metric, metric_kwargs=metric_kwargs)
 
 
+def test_argkminclassmode_factory_method_wrong_usages():
+    rng = np.random.RandomState(1)
+    X = rng.rand(100, 10)
+    Y = rng.rand(100, 10)
+    k = 5
+    metric = "euclidean"
```

Comment:
Here again I find it weird to test against `metric="euclidean"` since the specialization is not implemented at this time.